trch-bl0/vectors.s.gen

```diff
@@ -0,0 +1,363 @@
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+.word hang
+
+
+__entry: /* same as 'reset', but must not be marked with .thumb_func */
+
+.thumb_func
+reset:
+    b crt_init
+    b hang
+    b hang
+
+.thumb_func
+svc:
+    mov r0, #0
+    sub r0, #7 // 0xfffffff9: priveledged Thread mode with main stack
+    bx r0
+
+.thumb_func
+systick:
+    push {r0, r1, lr}
+
+    bl systick_isr
+
+    /* Clear Pending flag */
+    ldr r0, icsr_addr
+    mov r1, #1
+    lsl r1, #25 /* PENDSTCLR */
+    str r1, [r0]
+
+    pop {r0, r1, pc}
+
+    .align 2
+
+icsr_addr: /* re-useable by across exc handlers */
+    .word 0xe000ed04
+
+.thumb_func
+hang:   b .
+    b hang
+
+.thumb_func
+crt_init:
+    // Zero-initialize .bss
+    ldr r0, =__bss_start
+    ldr r1, =__bss_end
+    mov r2, #0
+bss_zero_loop:
+    str r2, [r0]
+    add r0, #4
+    cmp r0, r1
+    bne bss_zero_loop
+    bl _main
+    b hang
+
+
+.thumb_func
+exc2:
+    b exc2
+
+.thumb_func
+exc3:
+    b exc3
+
+.thumb_func
+exc4:
+    b exc4
+
+.thumb_func
+exc5:
+    b exc5
+
+.thumb_func
+exc6:
+    b exc6
+
+.thumb_func
+exc7:
+    b exc7
+
+.thumb_func
+exc8:
+    b exc8
+
+.thumb_func
+exc9:
+    b exc9
+
+.thumb_func
+exc10:
+    b exc10
+
+.thumb_func
+exc12:
+    b exc12
+
+.thumb_func
+exc13:
+    b exc13
+
+.thumb_func
+exc14:
+    b exc14
```

Comment:
This file (vectors.s.gen) is an automatically generated file and should not be included in the PR.  Please remove this file and confirm that it is still generated during the build.